learn/wme-backup-restore/backup/externalinstance/backup-esxi.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "Backup on ESXI"
+id: ""
+sidebar_label: "Backup on ESXI"
+---
+---
+
+
```

Comment:
VMware ESXi ESXi : not ESXI
Give some intro always.
for ex: On VMware ESXi we can take backups in two differnt ways. 
1. Take backup of directory from inside the Platform Instance
    - What was the command and process
2. Take snapshots using VMware ESXi console.
  - Explain the process first 
- for more info go to this link